cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -851,18 +851,18 @@ Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
 #   CLUSTER_IP_RANGE
 #   SERVICE_CLUSTER_IP_RANGE
 function Configure-CniNetworking {
-  $CNI_RELEASE_VERSION = 'v0.8.0'
+  $CNI_RELEASE_VERSION = 'v0.8.2-gke.0'
   if ((ShouldWrite-File ${env:CNI_DIR}\win-bridge.exe) -or
       (ShouldWrite-File ${env:CNI_DIR}\host-local.exe)) {
     $tmp_dir = 'C:\cni_tmp'
     New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
 
-    $release_url = ('https://github.com/containernetworking/plugins/releases/' +
-        'download/' + $CNI_RELEASE_VERSION + '/')
+    $release_url = ('https://www.googleapis.com/storage/v1/b/gke-release/o/cni-plugins%2f' +
```

Comment:
Why '%2f' instead of '/' ?

Reply:
it has to be encoded to be url safe. if using '/' directly here, the server would return with '404 not found'

Reply:
Ok - I'm not sure why the existing code seems to work <shrug>